plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -99,10 +100,10 @@ provider recorder plId ideState typ contents fp fo = withIndefiniteProgress titl
                         pure . Right $ InL $ makeDiffTextEdit contents out
                     ExitFailure n -> do
                         logWith recorder Info $ StdErr err
-                        pure . Left . responseError $ "Fourmolu failed with exit code " <> T.pack (show n)
+                        pure . Left . PluginInternalError $ "Fourmolu failed with exit code " <> T.pack (show n)
```

Comment:
another good place for `throwError` IMO

Reply:
This is actually a IO do block, so throwError won't work with PluginError here (And if it did we would end up throwing an IO exception...)